Islandora-Devops/isle-dc
# ISLE Sprint Work

Supersedes https://github.com/Islandora-Devops/isle-dc/pull/126 and https://github.com/Islandora-Devops/isle-dc/pull/123

This represents several tasks from the latest ISLE sprint and is ready for testing.
- Secrets
- Using a single variable for Domain
- Only one alias, which we've found is only neccessary for when you don't own a domain
- Better Drupal config for https and reverse proxy

This requires https://github.com/Islandora-Devops/isle-buildkit/pull/87 to test

### Testing Instructions

- Pull down and build https://github.com/Islandora-Devops/isle-buildkit/pull/87
- `cp sample.env .env` if you haven't already
- Set the following `.env`
```
USE_SECRETS=true
DOMAIN=islandora.traefik.me
REPOSITORY=local
```
- `make`
- `docker-compose up -d`

Everything should come up you can smoke test uploading content.  I've tested fcrepo indexing, derivatives, the openseadragon viewer, and matomo.  Everything seems to check out.
